native_store.go

```diff
@@ -120,6 +122,9 @@ func (ns *nativeStore) Put(ctx context.Context, serverAddress string, cred auth.
 		return err
 	}
 	_, err = ns.exec.Execute(ctx, bytes.NewReader(credJSON), "store")
+	if err != nil && err.Error() == errExeNotInPathMessage {
+		return fmt.Errorf("credential store is configured to `desktop` but docker desktop seems not running")
```

Comment:
Should we call it `credentials store` by following the consistent naming with Docker? See https://docs.docker.com/engine/reference/commandline/login/#credentials-store

Reply:
Will this error message `Error: error storing credentials: ` be added by ORAS to the beginning in this output?

Reply:
You can take a look at the screen shot above. It looks like the answer is no. We can change it if we need. @FeynmanZhou

Reply:
> Will this error message `Error: error storing credentials: ` be added by ORAS to the beginning in this output?

You can take a look at the screen shot above. It looks like the answer is no. We can change it if we need. @FeynmanZhou

Reply:
> Should we call it `credentials store` by following the consistent naming with Docker? See https://docs.docker.com/engine/reference/commandline/login/#credentials-store

Nice catch. Changed the name.

Reply:
@wangxiaoxuan273  You can check the log style in other ORAS error logs. If there is a consistent log style in ORAS, we may need to follow it and add it accordingly. If not, we can skip it.